src/app/shared/search/search-filters/search-filter/search-filter.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="facet-filter d-block mb-3 p-3" *ngIf="active$ | async"
-     [id]="regionId" [attr.aria-labelledby]="toggleId" [ngClass]="{ 'focus': focusBox }" role="region">
+ [id]="regionId" [attr.aria-labelledby]="toggleId"  [ngClass]="{ 'focus': focusBox }" role="region">
```

Comment:
You should be able to also undo all the changes in this file.  It appears you just removed the spaces from this file which made the code aligned better.